tests/conftest.py

```diff
@@ -110,13 +110,45 @@ def fixture_config():
         surface_layer_height = 0.1
 
         [plants]
-        a_plant_integer = 12
-        [[plants.ftypes]]
-        pft_name = "shrub"
-        max_height = 1.0
-        [[plants.ftypes]]
-        pft_name = "broadleaf"
-        max_height = 50.0
+        [[plants.pft_definition]]
```

Comment:
I assume these are variables that go like this into pyrealm, I would still encourage following our naming convention and make the names more descriptive/longer